docs/gangplank/usage.md

```diff
@@ -75,6 +75,13 @@ The following are optional commands:
 
 If `--workDir` is defined, the build output will be emited to `<workDir>/builds`.
 
+*btrfs warning*: Gangplank and CoreOS Assemlber are ridiciously slow on BTRFS filesystem. If you are building on btrfs (default for Fedora 33+), it is recommended that you turn off copy-on-write (COW) and use a `--workDir` with that directory if using parallel stages. Example:
```

Comment:
(Default for Fedora *Workstation* 33...not Fedora Server or FCOS)

But why do you say this?  Did you do side-by-side comparisons vs e.g. xfs?  I use brtfs on my workstation too (xref https://blog.verbum.org/2020/07/14/on-btrfs/ ) but I haven't noticed anything too bad lately, though I also haven't tried a side-by-side.

Reply:
Yeah, this was tested in a side-by-side comparison. When running multiple instances of COSA via Gangplank created Pods on `btrfs` it _crawls_ to a near halt during the clean-up phases. I have done several tests (with compression on/off) but the only way that multiple pods work correctly is when COW is turned off.